src/20_Components/amp-font.html

```diff
@@ -0,0 +1,95 @@
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-font" src="https://cdn.ampproject.org/v0/amp-font-0.1.js"></script>
+  <!-- 
+    Define some CSS classes so that `amp-font` can add or remove those based on whether a font was loaded or is in error-state. Find out about all the supported css classes [here](https://github.com/ampproject/amphtml/blob/b1607dff9446d332afcdead64b63e8d25e74cff7/extensions/amp-font/amp-font.md)
+  -->
+  <style amp-custom>
+    @font-face {
+      font-family: 'Font Does Not exist';
+      font-style: normal;
+      font-weight: 400;
+      src: url(fonts/FontDoesNotExist.ttf) format('truetype');
+    }
+    @font-face {
+      font-family: 'Tangerine';
+      font-style: normal;
+      font-weight: 400;
+      src: local('Tangerine'), url(http://fonts.gstatic.com/s/tangerine/v7/HGfsyCL5WASpHOFnouG-RJBw1xU1rKptJj_0jans920.woff2) format('woff2');
+      unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
+    }
+
+    .font-does-not-exist-loaded .font-does-not-exist{
+      font-family: 'Font Does Not exist', serif, sans-serif;
+    }
+    .font-does-not-exist-loading {
+      color: #00f;
+    }
+    .font-does-not-exist-loading .font-does-not-exist {
+      color: #0ff;
+    }  
+    .font-does-not-exist-missing .font-does-not-exist {
+      color: red;
+    }
+    .slabo-loaded .slabo-exist{
+      font-family: 'Tangerine', serif, sans-serif;
+    }
+    .slabo-loading {
+      color: #00f;
+    }
+    .slabo-loading .slabo {
+      color: #0ff;
+    }  
+    .slabo-missing .slabo {
+      color: red;
+    }
+  </style>
+</head>
+<body class="font-does-not-exist-loading">
+  <!-- #### Basic Usage -->
+  <!--
+    A simple text showing how to use `amp-font`.
+  -->
+  <p class="font-does-not-exist">
+    This text is displayed in red if the chosen font is not supported.
+  </p>
+  <p class="slabo">
+    This text is displayed in red if the chosen font is not supported.
+  </p>
+    <!--
+    Embed amp-font inside the body or the head of the document.
+    Use the attribute `timeout` to specify the time in milliseconds to wait until the font becomes available. This attribute is optional and it's default value is 3000.
+    Use `on-error-remove-class`, `on-error-add-class`, `on-load-remove-class`, `on-load-add-class` to specify the CSS classes to be used if the font loads correctly or not. For a detailed description,, please refer to the [documentation](https://www.ampproject.org/docs/reference/extended/amp-font.html)
+    -->
+  <amp-font
+    layout="nodisplay"
+    font-family="Font Does Not exist"
+    timeout="2000"
```

Comment:
Would be good to have an example with `timeout=0`. This is the coolest mode: "Load only if already in cache"